packages/react-native-room-kit/src/components/HMSLiveIndicator.tsx

```diff
@@ -37,6 +50,10 @@ const _HMSLiveIndicator = () => {
     },
   }));
 
+  const showParticipantList = () => {
+    setModalVisible(ModalTypes.PARTICIPANTS);
```

Comment:
```suggestion
      if (isHLSViewer) {
        setModalVisible(ModalTypes.PARTICIPANTS);
      } else {
        batch(() => {
          dispatch(setActiveChatBottomSheetTab('Participants'));
          dispatch({ type: 'SET_SHOW_CHAT_VIEW', showChatView: true });
        });
      }
```

also import required variables

Reply:
Done